### DISCUSSION TITLE
Formatter usable in CI?

### DISCUSSION THREAD

**Discussion by olix0r** [redhat-developer/vscode-yaml#839] [Q&A] [answered]
Is there a way to use the formatter in the VS Code extension from the CLI?

We can't rely on everyone who works in our repo to be using VS Code with this extension, and so I want to a way to (1) make it possible for these people to apply formatting as done by this extension and (2) in CI, I'd like to run a check that all YAML files are formatted consistently.

Is this currently possible? If not, is this something likely to be implemented if I open an issue? Or are we better off disabling the extension's formatter and using another tool for this?

**Comment by gorkem** [comment, marked as answer]
Yes, it is possible... sort of. `vscode-yaml` is using [prettier](https://prettier.io/) as the formatter, and I recommend using prettier directly on your CI.
